case-lib/lib.sh

```diff
@@ -741,29 +741,29 @@ is_ipc4()
 logger_disabled()
 {
     local ldcFile
-    # Some firmware/OS configurations do not support logging.
-    ldcFile=$(find_ldc_file) || {
-        dlogi '.ldc dictionary file not found, SOF logs collection disabled'
-        return 0 # 0 is 'true'
-    }
 
     # Disable logging when available...
-    if [ ${OPT_VAL['s']} -eq 0 ]; then
-        return 0
-    fi
+    [ ${OPT_VAL['s']} -eq 0 ] && return 0
```

Comment:
This is not needed and it looks incompatible with `set -e`. It's actually compatible with `set -e` but that's confusing.

It's also less readable for some people and distracting git churn in a too complex and too messy area of the code.